pkg/utils/otel.go

```diff
@@ -90,3 +154,32 @@ func newTraceProvider(ctx context.Context, version string) (*trace.TracerProvide
 	)
 	return traceProvider, nil
 }
+func logProvider(serviceName, version, endpoint, headers string) error {
+	_ = os.Setenv("OTEL_EXPORTER_OTLP_HEADERS", headers)
```

Comment:
```suggestion
	_ = os.Setenv("OTEL_EXPORTER_OTLP_ENDPOINT", "")
	_ = os.Setenv("OTEL_EXPORTER_OTLP_HEADERS", headers)
```
set it to empty, it will be read by the provider otherwise